app/models/supplejack_api/support/searchable.rb

```diff
@@ -37,7 +37,7 @@ def custom_find(id, scope = nil, options = {})
           options ||= {}
           class_scope = unscoped
           class_scope = class_scope.active unless options.delete(:status) == :all
-          column = "#{name.demodulize.downcase}_id"
+          column = "#{name.demodulize.downcase}_id".to_sym # .to_sym prevents possible SQL injection
```

Comment:
Obviously we don't have SQL in Mongo, but the point still stands ... fix `rubucop`

Reply:
rubocop or brakeman? ^^

Reply:
True Brakeman. I have to update due to `bundle audit` since I created the branch anyway so I'll just fix the comment